dhcpv4/server4/server.go

```diff
@@ -123,7 +123,7 @@ func WithConn(c net.PacketConn) ServerOpt {
 }
 
 // NewServer initializes and returns a new Server object
-func NewServer(addr *net.UDPAddr, handler Handler, opt ...ServerOpt) (*Server, error) {
+func NewServer(addr *net.UDPAddr, handler Handler, iface string, opt ...ServerOpt) (*Server, error) {
```

Comment:
Can this be renamed to `func NewServer(ifname string, addr *net.UDPAddr, handler Handler, opt ...ServerOpt) (*Server, error) {`

so that it matches the v6 one: https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/server6/server.go#L120